packages/suite/src/components/suite/CountdownTimer.tsx

```diff
@@ -1,31 +1,47 @@
 import React, { useEffect, useState } from 'react';
 import { useLocales } from '@suite-hooks/useLocales';
 import { formatTimeLeft } from '@suite-utils/formatTimeLeft';
+import { Translation } from './Translation';
 
 interface TimerProps {
     deadline: number | string;
     format?: Array<keyof Duration>;
+    isApproximate?: boolean;
     className?: string;
 }
 
 export const CountdownTimer = ({
     deadline,
     format = ['hours', 'minutes'],
+    isApproximate,
     className,
 }: TimerProps) => {
     const locale = useLocales();
 
     const [timeLeftString, setTimeLeftString] = useState(
         formatTimeLeft(new Date(deadline), locale, format),
     );
+    const [isPastDeadline, setIsPastDeadline] = useState(false);
 
     useEffect(() => {
         const interval = setInterval(() => {
+            setIsPastDeadline(new Date(deadline).getTime() <= Date.now());
```

Comment:
nit: 
Maybe instead of showing 0 seconds for 1 second. Would it make sense to show `Almost there...` right after 1 second?

```suggestion
            setIsPastDeadline(new Date(deadline).getTime() <= (Date.now() - 1000));
```

Reply:
Good idea!

Reply:
7e196ddd5

Reply:
If you want to set isPastDeadline on true 1 second BEFORE deadline, you should reduce the deadline timestamp or raise the current timestamp? Now you show 0 seconds for 2 seconds....
<img width="181" alt="image" src="https://user-images.githubusercontent.com/3729633/202437517-2b06fc8b-5b2c-4e67-845f-819de54a7698.png">

Reply:
Should be `+ 1000` 😬 

Force pushed the change